addons/website_slides/controllers/main.py

```diff
@@ -212,6 +216,13 @@ def slide_view(self, slide, **kwargs):
 
         values = self._get_slide_detail(slide)
         self._set_viewed_slide(slide)
+        # allow rating and comments
+        if slide.channel_id.allow_rating:
+            signed_token, partner_id = slide._get_signed_token(request.env.user.partner_id.id)
```

Comment:
Note that people viewing a slide cannot necessarily write on it :) . You should have a sudo somewhere otherwise I fear it will crash with regular people.